tuareg-opam.el

```diff
@@ -37,6 +37,23 @@
   "Support for the OPAM files."
   :group 'languages)
 
+;; TODO this is wrong, and doesn't respect OPAMROOT. It should probably just
+;; removed.
```

Comment:
Maybe we can make it right?  Querying opam is likely the right thing to do—do not remember why I did not choose that route.

Reply:
(I can no it later—no need to fix it for now.)

Reply:
To be honest, I would just disable all opam related functionality if an opam binary isn't set in the path. I don't think it's worth it to complicate the codebase for this use case.